agent/src/iconfucius/rasa/Makefile

```
@@ -0,0 +1,55 @@
+export RASA_LLM_MODEL           ?= claude-opus-4-6
+export RASA_REPHRASER_PROVIDER  ?= anthropic
+export RASA_REPHRASER_MODEL     ?= claude-opus-4-6
+export RASA_SUBAGENT_PROVIDER   ?= anthropic
+export RASA_SUBAGENT_MODEL      ?= claude-opus-4-6
+export RASA_SUBAGENT_TOOL_TIMEOUT ?= 120
+export ICONFUCIUS_MCP_URL          ?= http://127.0.0.1:5138/mcp
+export LANGFUSE_HOST               ?= http://localhost:12526
+export LANGFUSE_PUBLIC_KEY         ?= placeholder
+export LANGFUSE_PRIVATE_KEY        ?= placeholder
+
+.PHONY: rasa-inspect rasa-train rasa-train-for-release langfuse-up langfuse-down langfuse-logs
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)